server/src/main/java/org/elasticsearch/common/geo/BoundingBox.java

```diff
@@ -194,4 +195,9 @@ public int hashCode() {
     public String toString() {
         return "BBOX (" + left() + ", " + right() + ", " + top() + ", " + bottom() + ")";
     }
+
+    @Override
+    public TransportVersion getMinimalSupportedVersion() {
+        return TransportVersion.V_8_500_069;
+    }
```

Comment:
I think it'd be better to have this in the concrete subclasses. That way if we add another one in future we won't forget to set this to the correct value.

Also it (and the `getWriteableName` implementations) probably should to be `final` to make sure we don't accidentally add further subclasses without thinking about the consequences in this area..

Reply:
Done